core/unit_test/TestResize.hpp

```diff
@@ -132,8 +132,339 @@ void testResize() {
+    view_type view_3d("view_3d", sizes[0], sizes[1], sizes[2]);
+    const int* oldPointer = view_3d.data();
+    EXPECT_TRUE(oldPointer != NULL);
+    Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), view_3d,
+                   sizes[0], sizes[1], sizes[2]);
+    const int* newPointer = view_3d.data();
+    EXPECT_TRUE(oldPointer == newPointer);
+  }
+  {
+    typedef Kokkos::View<int****, DeviceType> view_type;
+    view_type view_4d("view_4d", sizes[0], sizes[1], sizes[2], sizes[3]);
+    const int* oldPointer = view_4d.data();
+    EXPECT_TRUE(oldPointer != NULL);
+    Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), view_4d,
+                   sizes[0], sizes[1], sizes[2], sizes[3]);
+    const int* newPointer = view_4d.data();
+    EXPECT_TRUE(oldPointer == newPointer);
+  }
+  {
+    typedef Kokkos::View<int*****, DeviceType> view_type;
+    view_type view_5d("view_5d", sizes[0], sizes[1], sizes[2], sizes[3],
+                      sizes[4]);
+    const int* oldPointer = view_5d.data();
+    EXPECT_TRUE(oldPointer != NULL);
+    Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), view_5d,
+                   sizes[0], sizes[1], sizes[2], sizes[3], sizes[4]);
+    const int* newPointer = view_5d.data();
+    EXPECT_TRUE(oldPointer == newPointer);
+  }
+  {
+    typedef Kokkos::View<int******, DeviceType> view_type;
+    view_type view_6d("view_6d", sizes[0], sizes[1], sizes[2], sizes[3],
+                      sizes[4], sizes[5]);
+    const int* oldPointer = view_6d.data();
+    EXPECT_TRUE(oldPointer != NULL);
+    Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), view_6d,
+                   sizes[0], sizes[1], sizes[2], sizes[3], sizes[4], sizes[5]);
+    const int* newPointer = view_6d.data();
+    EXPECT_TRUE(oldPointer == newPointer);
+  }
+  {
+    typedef Kokkos::View<int*******, DeviceType> view_type;
+    view_type view_7d("view_7d", sizes[0], sizes[1], sizes[2], sizes[3],
+                      sizes[4], sizes[5], sizes[6]);
+    const int* oldPointer = view_7d.data();
+    EXPECT_TRUE(oldPointer != NULL);
+    Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), view_7d,
+                   sizes[0], sizes[1], sizes[2], sizes[3], sizes[4], sizes[5],
+                   sizes[6]);
+    const int* newPointer = view_7d.data();
+    EXPECT_TRUE(oldPointer == newPointer);
+  }
+  {
+    typedef Kokkos::View<int********, DeviceType> view_type;
+    view_type view_8d("view_8d", sizes[0], sizes[1], sizes[2], sizes[3],
+                      sizes[4], sizes[5], sizes[6], sizes[7]);
+    const int* oldPointer = view_8d.data();
+    EXPECT_TRUE(oldPointer != NULL);
+    Kokkos::resize(Kokkos::view_resize(Kokkos::WithoutInitializing), view_8d,
+                   sizes[0], sizes[1], sizes[2], sizes[3], sizes[4], sizes[5],
+                   sizes[6], sizes[7]);
+    const int* newPointer = view_8d.data();
+    EXPECT_TRUE(oldPointer == newPointer);
+  }
```

Comment:
This block (above) is pretty much duplicated (only change is injection of `Kokkos::view_resize(Kokkos::WithoutInitializing)` as 1st parameter passed to `Kokkos::resize()`

Reply:
@dalg24. Thanks for your comments. In the new commit, I added tests in to CMakeLists.txt and combine the ```WithoutInitializing``` tests with the default ```resize``` tests.